### ISSUE TITLE
Analytics New KPI dashboard request [Pension - 527EZ]

### ISSUE THREAD

**Issue by buddhima-coforma** [department-of-veterans-affairs/va.gov-team#73162]
# Request a New Standardized KPI Dashboard or an Update to an Existing Dashboard
## What this form is for

Use this ticket template to request a new standardized KPI dashboard or an update to an existing KPI dashboard on Domo for your product. 

**Tell us what you need done:**
- [x] [New Dashboards](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/analytics-request-kpi-dashboard.md#for-new-dashboards) 

**This is for the re-launch of the pension benefits form. A dashboard similar to the 526 form would be great to have. 
Additionally, nice-to-have KPIs have been mentioned in the 'potential metrics' sheet.**

Standardized dashboards are currently available for [Forms](https://va-gov.domo.com/page/447193050?userId=853813555), [Content](https://va-gov.domo.com/page/426422632), & [Search](https://va-gov.domo.com/page/1964748112) products only. To see if you should request a standardized dashboard, check [Should I request a standardized KPI dashboard](https://docs.google.com/presentation/d/16mRcslbkWtW44rlsfQGXZnHtQ3GoLjn6GpFnd3Gvva8/edit#slide=id.p). If you’re not sure which dashboard(s) your product(s) qualify for, check the [Eligibility Dashboard](https://va-gov.domo.com/page/1550814739).

If your VA.gov product is not eligible for a standardized dashboard, you may be able to request an “edge case” dashboard. Please reach out to the Platform Analytics & Insights team on #vfs-platform-support for more information on this process.

### Expected delivery timelines

The Analytics team will notify you once this ticket has been pulled into a sprint. 

## For New Dashboards

In order to pull the correct data for your product, we need your help to define what pages and events to include, in addition to basic product info. We’ll need a filled out [KPI request template](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988) before we can pull this ticket into our sprint.

- [x] Make a copy of the [Google Sheet](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988)
- [x] Complete worksheet as instructed
- [x] Link your completed spreadsheet here: (https://docs.google.com/spreadsheets/d/1hopUPs_LcfXO8KHceJjYQD5ZKNSgnw-Ujefe_Ik8j3E/edit?usp=sharing)
- [x] Check your sharing settings to make sure the Analytics team will be able to access it
- [x] **Add your team label to follow progress**

**Additional Info For Both Options**

- Please use the [KPI Framework](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/Analytics%20Playbook/va-gov-platform-analytics-kpi-framework.pdf) for more information on the metrics used in the Standard KPI dashboards. 
- If you need any analytics support, read through the [How to request Analytics support](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Analytics-customer-support-guidelines.1586823275.html#Analyticscustomersupportguidelines-4.RequestaKPIDashboardinDomo).
- If you do not have Domo access, please [request access here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+access-request&template=analytics-request-google-analytics-domo-access.md&title=Request+access+to+Google+Analytics+and%2For+Domo).

**Comment by michelle-dooley** [comment]
Thanks @vbellissimo - we will review and get back to you with any questions. Should be able to pick this up in the next sprint.

**Comment by vbellissimo** [comment]
Thank you @michelle-dooley !

**Comment by buddhima-coforma** [comment]
@michelle-dooley thanks! FYI - I've listed some ideas the spreadsheet in a different tab (Potential Metrics). I'd like to check in with you on what might be possible there. Happy to get on a call and discuss.

**Comment by michelle-dooley** [comment]
@buddhima-coforma - could you please provide the full url  (like you did Applicant Information) for each of the steps in the spreadsheet not just the end/sub-string. The format must be the same as other form funnels on standardized dashboard in order for us to be able to add it.

**Comment by Blivaditis** [comment]
Hello, when are you planning for this form to go live? I need a start date for the form in order to pull the data.

**Comment by buddhima-coforma** [comment]
@Blivaditis We'll open up the form in production incrementally (using flipper), aiming to start with 25% of traffic on 1/30.

**Comment by Blivaditis** [comment]
Great thank you for the quick response!

**Comment by johnny-jesensky-adhoc** [comment]
Hi @buddhima-coforma, we've gone ahead and added the 527ez form to the [Forms KPI dashboard](https://va-gov.domo.com/page/447193050). Everything seems to have been configured properly, though it looks like there may be some test data in there. 

<img width="1383" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBekFLQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--eed0663d6f23067f278f2406e39ddd70639268a7/image.png" alt="image.png" />


Please let us know if you need anything else on this front.

FYI @vbellissimo, @Blivaditis, @dorseypa1

**Comment by buddhima-coforma** [comment]
Thanks @johnny-jesensky-adhoc!

I'm curious if we are we able to get metrics such as:
- How much time are they spending on pages? (Which pages are they spending most/least amount of time on?)
- Which pages do they start navigating back from, when they return?
- Session durations/time to complete a form
- Authenticated vs unauthenticed users
- Any errors users face
- Rage clicking

Let me know if it's possible and what we need to do? Thanks again!

**Comment by johnny-jesensky-adhoc** [comment]
Hi @buddhima-coforma, apologize for the delay on this. I spoke this over briefly with someone on our team yesterday. Some of these additional metrics might be tough to track, though it might be easier to provide details as to _why_ over a call.

You are welcome to join our _Office Hours_ session next week (Tues, Thurs, 3:30pm ET), where can give you more details and talk through these.

**Comment by buddhima-coforma** [comment]
@johnny-jesensky-adhoc I was out on PTO. Will pop in at office hours this week.
